src/Mailer/Transport/AwsSesTransport.php

```diff
@@ -21,7 +21,9 @@ class AwsSesTransport extends AbstractTransport
         'region' => 'us-east-1',
         'version' => 'latest',
         'aws_access_key_id' => '',
-        'aws_access_secret_key' => ''
+        'aws_access_secret_key' => '',
+        'timeout' => 0,
+        'connect_timeout' => 0,
```

Comment:
The value 0 of `connect_timeout` is actually interpreted as 300 seconds by curl.
https://curl.se/libcurl/c/CURLOPT_CONNECTTIMEOUT_MS.html

If we do not set `connect_timeout`, Guzzle sets the default 150 seconds.
https://github.com/guzzle/guzzle/blob/7.3.0/src/Handler/CurlFactory.php#L227

I think 150 is better because it does not change the default behavior.

```suggestion
        'connect_timeout' => 150,
```

0 of `timeout` is OK.

Reply:
Done! (sorry for the delay)